snowflake/README.md

```diff
@@ -110,7 +110,16 @@ datadog-agent integration install datadog-snowflake==2.0.1
     Snowflake metrics are aggregated by day, you can increase the interval to reduce the number of queries.<br>
     <bold>Note</bold>: Snowflake ACCOUNT_USAGE views have a <a href="https://docs.snowflake.com/en/sql-reference/account-usage.html#data-latency">known latency</a> of 45 minutes to 3 hours.</div>
```

Comment:
Use markdown instead of HTML

Reply:
markdown links won't render in this `div` alert box